b2b-sample/guardio-insurance-sample-application/util/apiDecode/settings/role/decodePatchRole.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * Copyright (c) 2022, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import callPatchRole from
+    "../../../apiCall/settings/role/callPatchRole";
+import { commonDecode } from "../../../util/apiUtil/commonDecode";
+import { PatchMethod } from "../../../util/common/common";
+
+function getAddReplaceBody(patchMethod, path, value) {
+    return {
+        "operations": [
+            {
+                "op": patchMethod,
+                "path": path,
+                "value": value
+            }
+        ]
+    }
+}
+
+function getRemoveBody(patchMethod, path, value) {
+    return {
+        "operations": [
+            {
+                "op": patchMethod,
+                "path": `${path}[value eq ${value}]`
+            }
+        ]
+    }
+}
+
+function getPatchBody(patchMethod, path, value) {
+    switch (patchMethod) {
+        case PatchMethod.ADD:
+
+            return getAddReplaceBody(patchMethod, path, value);
+
+        case PatchMethod.REPLACE:
+
+            return getAddReplaceBody(patchMethod, path, value);
+        case PatchMethod.REMOVE:
+
+            return getRemoveBody(patchMethod, path, value);
+    }
+}
+
+/**
+ * 
```

Comment:
Shall we add function descriptions. Check other places as well.

Reply:
Fixed with https://github.com/wso2/samples-is/pull/416/commits/732b9bcc7aa8badcd42dd91959331576e9a105ab